lib/web/integrations_awsoidc.go

```diff
@@ -302,21 +312,33 @@ func extractAWSRegionsFromQuery(r *http.Request) ([]string, error) {
 	return ret, nil
 }
 
+// regionsForListingDeployedDatabaseService fetches relevant aws regions, parse the regions query param, and returns the overlap.
```

Comment:
```suggestion
// regionsForListingDeployedDatabaseService fetches relevant AWS regions, parses the regions query param, and returns the overlap.
```

Reply:
Also, I would add a sentence here explaining why we're doing this intersection - the PR description explains it pretty well but reading the code without the context might be difficult to understand the reason.